pkg/server/handler/v1alpha1/resource.go

```diff
@@ -1,37 +1,38 @@
-package handler
+package v1alpha1
```

Comment:
Handler also has version, I think it can be with descriptors. The folder structure may be:

```
├── pkg
│   ├── apis
│   │   └── cyclone
│   │       └── v1alpha1
│   ├── common
│   │   ├── client.go
│   │   └── signals
│   │       └── signal.go
│   ├── k8s
│   │   ├── clientset
│   │   │   ├── clientset.go
│   │   │   ├── doc.go
│   │   │   ├── fake
│   │   │   ├── scheme
│   │   │   └── typed
│   │   ├── informers
│   │   │   ├── cyclone
│   │   │   ├── factory.go
│   │   │   └── generic.go
│   │   └── listers
│   │       └── cyclone
│   ├── server
│   │   ├── README.md
│   │   ├── apis
│   │   │   ├── api.go
│   │   │   ├── filters
│   │   │   ├── middlewares
│   │   │   ├── v1alpha1
│   │   │   │   ├── descriptors
│   │   │   │   └──  handlers
│   │   │   ├── v1alpha2
│   │   │   │   ├── descriptors
│   │   │   │   └──  handlers
│   │   │   ├── api.go
│   │   ├── config
│   │   │   └── config.go
```

Reply:
Agreed with versioned handlers